x/lightclient/keeper/msg_server_update_verification_flags.go

```diff
@@ -0,0 +1,33 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authoritytypes "github.com/zeta-chain/zetacore/x/authority/types"
+	"github.com/zeta-chain/zetacore/x/lightclient/types"
+)
+
+// UpdateVerificationFlags updates the light client verification flags.
+// This disables/enables blocks verification of the light client for the specified chain.
+// Emergency group can disable flags while operation group can enable/disable
+func (k msgServer) UpdateVerificationFlags(goCtx context.Context, msg *types.MsgUpdateVerificationFlags) (
+	*types.MsgUpdateVerificationFlagsResponse,
+	error,
+) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	requiredGroup := authoritytypes.PolicyType_groupEmergency
+	if msg.VerificationFlags.EthTypeChainEnabled || msg.VerificationFlags.BtcTypeChainEnabled {
+		requiredGroup = authoritytypes.PolicyType_groupOperational
+	}
```

Comment:
NIT: We can move this to a separate function and add unit tests to it

Reply:
c2c0409c